### ISSUE TITLE
Use native js `bigint` rather than `number`

### ISSUE THREAD

**Issue by zone117x** [hirosystems/clarinet#82]
Looks like 128-bit Clarity integers are being represented as js `number`'s (64-bit floating point). This causes problems with numbers that take ~53 or more bits to represent. I think the best solution here is to use native js `bigint` to represent these values. 

IMO, helper constructors like 
https://github.com/hirosystems/clarinet/blob/9beb61a538be055c220886300705eac552f6bf28/deno/index.ts#L318-L320
should continue to accept `number` as well as `bigint` (principal of being most permissive for inputs, but most explicit for outputs):
```ts
function uint(val: number | bigint) {
  return `u${val}`;
}
```

But response values (that represent 128-bit ints/uints) should return a `bigint`. That would result in this example code
https://github.com/hirosystems/clarinet/blob/9beb61a538be055c220886300705eac552f6bf28/examples/billboard/tests/billboard_test.ts#L34-L35
becoming
```ts
        assetMaps = chain.getAssetsMaps();
        assertEquals(assetMaps.assets['STX'][wallet_1.address], balance - 210n);
```

Obviously this isn't much of a concern with something like a STX balance that is unlikely to be higher than a 53-bit integer, but FTs and many other use cases use larger numbers where this becomes an issue.

**Comment by lgalabru** [comment]
Thanks for looking into this @zone117x!! The current clarinet js library is pretty basic, and we should definitely upgrade the game there at some point!

**Comment by lgalabru** [comment]
Addressed with #148
